gradle/libs.versions.toml

```diff
@@ -158,6 +159,7 @@ dagger-hilt-android = { group = "com.google.dagger", name = "hilt-android", vers
 dagger-hilt-android-compiler = { group = "com.google.dagger", name = "hilt-android-compiler", version.ref = "dagger" }
 dagger-hilt-android-testing = { group = "com.google.dagger", name = "hilt-android-testing", version.ref = "dagger" }
 dagger-hilt-compiler = { group = "com.google.dagger", name = "hilt-compiler", version.ref = "dagger" }
+desugar-jdk-libs = { group = "com.android.tools", name = "desugar_jdk_libs", version.ref ="androidDesugar" }
```

Comment:
Minor (🔍): Consider adding the `android` prefix to this dependency, just like you added it on the version itself.

Reply:
Fixed in this [81a56c6](https://github.com/wordpress-mobile/WordPress-Android/pull/21029/commits/81a56c6afd858d484427b5709447e451c1f5ae18)